api-extractor/package.json

```diff
@@ -25,6 +25,7 @@
     "@types/z-schema": "3.16.31",
     "fs-extra": "~0.26.7",
     "jju": "~1.3.0",
+    "typescript": ">=2.2.1-0 <3.0.0-0",
```

Comment:
This was missing... api-extractor requires typescript, but was not including it in dependencies.